### DISCUSSION TITLE
apollo源码打包后启动登录时报ttpRequestMethodNotSupportedException: Request method 'POST' not supported]

### DISCUSSION THREAD

**Discussion by gy448** [apolloconfig/apollo#3771] [Q&A]
apollo源码打包后启动登录时报ExceptionHandlerExceptionResolver : Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'POST' not supported]
![image](https://user-images.githubusercontent.com/48306358/122722067-e6495580-d2a3-11eb-84f0-0859215c8337.png)
![image](https://user-images.githubusercontent.com/48306358/122722188-ff520680-d2a3-11eb-93c3-d2a2c5a5e148.png)

**Comment by jujunchen** [comment]
我也碰到了这个问题，我是基于2.1升级了spring security版本到5.7.8
